ui/src/views/Project.vue

```diff
@@ -59,5 +71,27 @@ export default class ProjectView extends Vue {
       }
     }
   }
+
+  copyProjectURI () {
+    const inputComponent = this.$refs.uriInput as Vue
+    const inputElement = inputComponent?.$el.querySelector('input')
+
+    if (!inputElement) { return }
+
+    inputElement.select()
+    const success = document.execCommand('copy')
+
+    if (success) {
+      this.$buefy.toast.open({
+        message: 'The project URI was copied to your clipboard',
+        type: 'is-success'
+      })
+    } else {
+      this.$buefy.toast.open({
+        message: 'Error: could not copy project URI',
```

Comment:
I almost think that maybe we don't need to show anything. But I guess it's good to be explicit in case people expect the copy to work. A warning maybe would be enough?

Reply:
Isn't it technically an error though? I can change it to a warning, that's fine as well

Reply:
The user can still Ctrl+C. I thought an error to something more definite

Reply:
But if they hit the button, they need to know that something happened. Otherwise they're gonna end up doing CTRL+V somewhere else and being frustrated that their clipboard doesn't contain the project URI

Reply:
Agree, I do not say to remove the message completely

Reply:
I changed the message and made it a warning